### ISSUE TITLE
In-compatible with AndroidX

### ISSUE THREAD

**Issue by zubairehman** [andrewackerman/flutter_permissions_helper#1]
Hi, 
i am getting this exception when i run my app:

```
import android.support.v4.app.ActivityCompat;
                             ^
  symbol:   class ActivityCompat
  location: package android.support.v4.app
/Users/zubairehman/.pub-cache/hosted/pub.dartlang.org/flutter_permissions_helper-0.0.4/android/src/main/java/com/andrewackerman/flutterpermissionshelper/FlutterPermissionsHelperPlugin.java:11: error: package android.support.v4.content does not exist
import android.support.v4.content.ContextCompat;
                                 ^
/Users/zubairehman/.pub-cache/hosted/pub.dartlang.org/flutter_permissions_helper-0.0.4/android/src/main/java/com/andrewackerman/flutterpermissionshelper/FlutterPermissionsHelperPlugin.java:199: error: cannot find symbol
        ActivityCompat.requestPermissions(activity, new String[] { manifest }, 0);
        ^
  symbol:   variable ActivityCompat
  location: class FlutterPermissionsHelperPlugin
/Users/zubairehman/.pub-cache/hosted/pub.dartlang.org/flutter_permissions_helper-0.0.4/android/src/main/java/com/andrewackerman/flutterpermissionshelper/FlutterPermissionsHelperPlugin.java:204: error: cannot find symbol
        int status = ContextCompat.checkSelfPermission(activity, manifest);
                     ^
  symbol:   variable ContextCompat
  location: class FlutterPermissionsHelperPlugin
/Users/zubairehman/.pub-cache/hosted/pub.dartlang.org/flutter_permissions_helper-0.0.4/android/src/main/java/com/andrewackerman/flutterpermissionshelper/FlutterPermissionsHelperPlugin.java:286: error: cannot find symbol
            if (ActivityCompat.shouldShowRequestPermissionRationale(registrar.activity(), permission)) {
                ^
  symbol:   variable ActivityCompat
  location: class FlutterPermissionsHelperPlugin
/Users/zubairehman/.pub-cache/hosted/pub.dartlang.org/flutter_permissions_helper-0.0.4/android/src/main/java/com/andrewackerman/flutterpermissionshelper/FlutterPermissionsHelperPlugin.java:289: error: cannot find symbol
                if (ActivityCompat.checkSelfPermission(registrar.context(), permission) == PackageManager.PERMISSION_GRANTED) {
                    ^
  symbol:   variable ActivityCompat
  location: class FlutterPermissionsHelperPlugin
6 errors
*******************************************************************************************
The Gradle failure may have been because of AndroidX incompatibilities in this Flutter app.
See https://goo.gl/CP92wY for more information on the problem and how to fix it.
*******************************************************************************************
Finished with error: Gradle task assembleDebug failed with exit code 1
```

**Comment by tmasunda** [comment]
I am getting the same error. has anyone managed to deal with this?

**Comment by andrewackerman** [comment]
AndroidX compatibility should be fixed by 58e9ee0ff6ac78811a7a6c6b1f05908a5d2bdd38.
